client/src/components/JurisdictionAdmin/BatchRoundDataEntry.tsx

```diff
@@ -1,606 +1,644 @@
-/* eslint-disable jsx-a11y/no-autofocus */
-import React, { useState } from 'react'
+import classnames from 'classnames'
```

Comment:
The changes to this file are so substantial it's probably easier to review the new code in isolation instead of the diff, hence the delete commit followed by a new file commit: https://github.com/votingworks/arlo/pull/1644/commits/5d812b985823e55dde6ae1370e32a684feb14f12